net/core/filter.c

```diff
@@ -74,6 +74,12 @@
 #include <net/ipv6_stubs.h>
 #include <net/bpf_sk_storage.h>
 
+#ifdef CONFIG_XDP_LUA
+#include <lua.h>
+#include <luadata.h>
+#include <luaunpack.h>
```

Comment:
one example of what I was arguing on slack is to have one single header to xdplua, e.g., `xdplua.h`